config.json

```diff
@@ -244,7 +244,10 @@
     "restrict_to_login": false,
     "swatch_option_size": "22x22",
     "social_icon_placement_top": false,
-    "social_icon_placement_bottom": "bottom_none"
+    "social_icon_placement_bottom": "bottom_none",
+    "show_geotrust_ssl_seal": false,
+    "geotrust_ssl_seal_hostname": "",
```

Comment:
Good question, however `shopPath` or `ShopPathSSL` return a url `http://...` not the hostname. But we could return the hostname in the `Setting` resource and use that